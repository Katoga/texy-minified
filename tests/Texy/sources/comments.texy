/--html
<p>
<!------->
\--


První <!-- komentář -->  <!--[ komentář2 ]-->  <!-- komentář3 < > -->

<!-- invalid -- comment1 -->

<!-- invalid -- comment2---->

<!---- invalid -- comment2---->
