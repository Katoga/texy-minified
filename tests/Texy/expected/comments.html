<p><!-- - --></p>

<p>První<!-- komentář --> <!--[ komentář2 ]--><!-- komentář3 < > --></p>
<!-- invalid  -  comment1 --><!-- invalid  -  comment2 - --><!-- -  invalid  -  comment2 - -->
